instrumentation/opentelemetry-instrumentation-openai/test-requirements.txt

```diff
@@ -0,0 +1,8 @@
+openai==1.37.1
```

Comment:
You need to add all the openai dependencies to keep things reproducible

Reply:
thanks! will do now

Reply:
At least pydantic should go here